neurom/apps/morph_stats.py

```diff
@@ -112,7 +122,13 @@ def extract_dataframe(neurons, config):
     if 'neuron' in config:
         del config['neuron']
 
-    stats = {nrn.name: extract_stats(nrn, config) for nrn in neurons}
+    func = partial(_run_extract_stats, config=config)
+    if n_workers == 1:
+        stats = dict(map(func, neurons))
+    else:
+        with multiprocessing.Pool(n_workers) as pool:
+            stats = dict(pool.imap_unordered(func, neurons))
+
     columns = list(next(iter(next(iter(stats.values())).values())).keys())
```

Comment:
0_o

Reply:
(0(o_o(ooo)a_a)A)

Reply:
Aleksei had more or less the same reaction.

Reply:
@asanin-epfl You are a wise man.